tests/html_angular/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -7490,7 +7490,9 @@ can be found in the LICENSE file at http://angular.io/license
   Binding</a
 ><br />
 <div
-  style="margin-left:8px"
+  style="
+    margin-left: 8px;
+  "
```

Comment:
I think this should be in one line, we fit in a line

Reply:
`printWidth: 1` test... haha